stories/atoms/components/button/Button.stories.mdx

```diff
@@ -31,196 +28,343 @@ import { Icon } from 'ui/atoms/icon/Icon'
 
 # Button
 
-> Communicate _actions_ that users can take.
+> Communicates actions that users can take.
 
 [![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
 
-## Overview
+## Description
 
-### Description
+Buttons are mainly used to give the user options to act. They help users to find
+the most important actions through the UI and enable them to perform these actions.
 
-Buttons are mainly used to show the user’s choice of options for actions. A button helps the users to find
-the most important actions throughout UI and enables them to perform these actions.  
-By using one or more of the **content** properties it's possible to create appealing and intuitive buttons
+By using one or more of the button properties, it's possible to create appealing and intuitive buttons
 that will help users to find the action they're looking for.
 
----
+## Overview
 
 <Story
   name="Overview"
   argTypes={{
-    icon: { control: false, table: { type: { summary: 'JSX.Element' } } },
-    leftIcon: { control: false, table: { type: { summary: 'JSX.Element' } } },
-    rightIcon: { control: false, table: { type: { summary: 'JSX.Element' } } }
+    icon: { control: false, table: { type: { summary: 'Node' } } },
+    leftIcon: { control: false, table: { type: { summary: 'Node' } } },
+    rightIcon: { control: false, table: { type: { summary: 'Node' } } }
   }}
   args={{
     label: 'Connect wallet',
-    icon: <Icon name="wallet" size={22} />
+    variant: 'primary',
+    size: 'medium',
+    backgroundColor: 'primary',
+    disabled: false
   }}
 >
-  {args => (
-    <div
-      style={{
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'center',
-        gap: '25px'
-      }}
-    >
-      <Button {...args} />
-    </div>
-  )}
+  {args => {
+    const { theme } = useTheme()
+    const invert = theme === 'light'
+    return (
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+          gap: '25px'
+        }}
+      >
+        <Button {...args} icon={<Icon name="wallet" size={22} invertColor={invert} />} />
+      </div>
+    )
+  }}
 </Story>
 
----
-
-### Properties
+## Properties
 
 <ArgsTable story="Overview" />
 
+## Label
+
+The `label` is the text description of Button that help communicate what happens after the click.
+If Button is an **icon** `variant`, the `label` is not shown, however it is still there as a title
+attribut used for the [tooltip](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role)
+(as you can see by hovering your mouse over the icon button below).
+
+The `label` can have any length however Button width will never exceed 273px.
+
+<Canvas>
+  <Story name="Label">
+    {() => {
+      const { theme } = useTheme()
+      const invert = theme === 'light'
```

Comment:
If we want to be consistent about the naming of boolean variables, I would have named it:
```suggestion
      const isLightTheme = theme === 'light'
```